src/libcore/option.rs

```diff
@@ -10,9 +10,9 @@
 
 //! Optional values
 //!
-//! Type `Option` represents an optional value: every `Option`
+//! Type `Option<T>` represents an optional value: every `Option<T>`
```

Comment:
I asked @steveklabnik about this on IRC and he was in favor of adding them everywhere. Quoting from the IRC conversation at https://botbot.me/mozilla/rust/2015-03-27/?msg=35229748&page=24:


> jviereck: The current docs use "option" and "Option" and "Option<T>" when the type is relevant. Maybe all of the options should be renamed to `Option` to make clear the trait is meant and have a consistent wording?
> steveklabnik: it should always be `Option<T>`